admin_guide/diagnostics_tool.adoc

```diff
@@ -298,6 +298,11 @@ A user-defined timeout may be set by passing the
 if a newly-created log entry is not able to be queried via Elasticsearch after
 45 seconds.
 
+|`sdn`
+|This check performs some cluster-level diagnostics of the OpenShift SDN.  In particular, it verifies that master hosts can connect to kubelets and that nodes can route packets to each other; validates node addresses and HostSubnet objects; and gathers networking-related objects from the OpenShift API and logs, OVS flows, iptables rules, and other network configuration from master and node hosts that may help to diagnose problems with the SDN.
+
+This Ansible-based check complements the `oc adm diagnostics networkcheck` diagnostic: the former is oriented towards diagnosing problems at the infrastructure or node level whereas the latter is oriented towards diagnosing problems at the application or pod level.  Use `oc adm diagnostics networkcheck` when the master is able to schedule pods on nodes but pods have connectivity problems.  Use the Ansible check if the cluster is in a state such that diagnostic pods cannot be scheduled or do not provide sufficient information to troubleshoot a problem.
```

Comment:
I might replace this paragraph with "This check can help you diagnose pod or infrastructure problems, particularly if you cannot schedule diagnostic pods or they do not provide enough information to troubleshoot the issue."

At line 118, I'd say, "Create diagnostic pods on multiple nodes to diagnose common network issues from an application or pod standpoint. Run this diagnostic when when the master can schedule pods on nodes, but the pods have connection issues. This check confirms that pods can connect to services, other pods, and the external network."

Reply:
Changes made.  Thanks!